### PR TITLE
use rfc1918 ip addresses for default values

### PR DESCRIPTION
**Pull request by furlongm**
Tested and confirmed working locally.

### CODE REVIEW

**Comment by kamidzi** [comment]
Please provide some output for your testing.

File: cookbooks/bcpc/attributes/default.rb [hunk 1]
```diff
@@ -157,9 +157,9 @@
 # to be set properly
 default['bcpc']['floating']['interface-parent'] = nil
 
-default['bcpc']['fixed']['cidr'] = "1.127.0.0/16"
+default['bcpc']['fixed']['cidr'] = "10.17.16.0/20"
```
Comment (kamidzi) [inline cookbooks/bcpc/attributes/default.rb]: Is there reason to use a smaller network?

Comment (furlongm) [reply cookbooks/bcpc/attributes/default.rb]: The larger the network, the greater the risk of overlap in rfc1918 address space. We don't actually need a /20 for testing (which is where this value is used mostly).

**Comment by kelvk** [comment]
This will impact population of DNS records for fixed network. See #1202. It still is assumed to be classful.

**Comment by furlongm** [comment]
"Current DNS zone configuration assumes float subnets are classful (/8, /16) and this could result in incorrect DNS resolution on VMs." - is this something that could be fixed in the DNS zone configuration?

**Comment by kelvk** [comment]
Yes. https://github.com/bloomberg/chef-bcpc/blob/master/cookbooks/bcpc/recipes/powerdns.rb#L121 is probably the place to start.

**Comment by bcpc** [comment]
@furlongm #1280 rebased on master with singlehead: is successful! click https://cloud-infra.jaas.dev.bloomberg.com/job/bcpc/job/jenkins/job/master/177/

**Pull request closed without merging**